.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,231 @@
+This document defines many guidelines that should be adhered to when developing 
+against Banshee. These guidelines will make the codebase more readable,
+extensible, and portable.
+
+Reporting Issues
+================
+
+[GNOME's bugzilla](https://bugzilla.gnome.org/buglist.cgi?quicksearch=product%3Abanshee)
+is the official issue tracker and contributors are encouraged to file bugs and
+feature requests there in addition to this repository.
```

Comment:
The wording in this document was already pretty heavy handed (_adhered to_ above and _violation_ of coding guidelines below) so I wanted to soften this a bit - hence _encouraged_ here.

I wanted to highlight that we're trying to provide an adapter on Banshee's cathedral development model with our repositories but I'm not sure this is the best place to do it - the file's already quite long - so I'm considering extending this pull request with a `README.md` where we can say that and link to the original Banshee `README` for further information.

What do you think?